pkg/source/template.go

```diff
@@ -20,6 +21,18 @@ func (i InvalidPluginSpecError) Error() string {
 	return i.err
 }
 
+// for backward compatibility
```

Comment:
Why is that important for backward compat?

Reply:
because by default krew-release-bot added the `indent` of 4 spaces before `sha256`, but if someone want it to be 6, they would instead get 10 (because we didn't change addURIAndSha for backward compatibility), so fixShaIndentation removes those extra 4 spaces.

added more tests for this .